charts/pulsar-detector/values.yaml

```diff
@@ -0,0 +1,59 @@
+# Default values for pulsar-detector.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+namespace: "snpe4"
+cluster: "pulsar4-sn-platform"
+metadataPrefix: ""
+
+auth:
+  authentication:
+    enabled: true
+    brokerClientAuthenticationParameters: NmM3MGIwZTUtMDQ3My1jZDc1LTNmNDctMDUwNWM0NTFmOWVhOjkyNDllYTNhLTUyMjEtMDJkNS1iODY2LTFjYmUzNmQ1ZTYxNAo=
```

Comment:
auth should be considered to be added to the common module and use secret to save it